README.md

```diff
@@ -1,53 +1,24 @@
-# The `Hello Rails React`
+# README
 
+This README would normally document whatever steps are necessary to get the
+application up and running.
 
-> Description: .... pending
+Things you may want to cover:
 
-### Build with:
+* Ruby version
 
-- React JS
-- Node JS
-- RoR
+* System dependencies
 
-## Preparation
+* Configuration
 
-- Install [Node.js](https://nodejs.org/en/).
-- To check, if Node.js is installed already, then just check Node.js and npm versions: `node -v` and `npm -v`
-- Instal `React Dev tools` in Google extentions. (we need it for future)
+* Database creation
 
-## Getting Started
+* Database initialization
 
-- git clone https://github.com/elmar8287/hello-rails-react.git
-- cd ./hello-rails-react
```

Comment:
- [ ] I would recommend making the README file professional. It should have a project title, description, all the steps to clone and run the project locally, author details so that anyone can understand and run your project. You can follow the [microverse README template](https://github.com/microverseinc/readme-template).

Reply:
Fixed